pkg/broker/ingress/handler_test.go

```diff
@@ -295,7 +298,11 @@ func TestHandler(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			reportertest.ResetIngressMetrics()
 			ctx := logging.WithLogger(context.Background(), logtest.TestLogger(t))
-			ctx, cancel := context.WithTimeout(ctx, 3*time.Second)
+			timeout := 3 * time.Second
```

Comment:
As discussed, increasing this default might reduce the flakiness in the future as we have more tests closed to 3s. Maybe change it to 5?

Reply:
Yeah, makes sense, given recent observations (slightly above 3 seconds even for the HealthCheck test). Have increased to 5 for the rest.